util/chplenv/chpl_home_utils.py

```diff
@@ -17,6 +17,11 @@ def get_chpl_home():
         chpl_home = dirname(dirname(dirname(os.path.realpath(__file__))))
     return chpl_home
 
+@memoize
+def get_chpl_third_party():
```

Comment:
Follow-up Q: Do we need one of these for `get_chpl_runtime()`, `get_chpl_compiler()`, etc.?